package.json

```diff
@@ -232,9 +239,13 @@
     "test:shared:ci": "npm run build:assets && jest --config shared/jest-shared.config.ts --coverageDirectory=./coverage/${NODE_INDEX} ${TESTFILES}",
     "test:shared:file": "npm run build:assets && jest --config shared/jest-shared.config.ts --coverage=false",
     "test:watch": "jest --watch .*test\\.js$",
+    "postgres:run": "cd web-api/src/persistence/postgres && docker compose up -d",
     "verify-authorizers": "./scripts/circleci/verify-authorizers.sh",
     "verify-private-elasticsearch": "./scripts/circleci/verify-private-elasticsearch.sh",
-    "visualize-bundle": "esbuild-visualizer --metadata ./metadata.json && open stats.html"
+    "visualize-bundle": "esbuild-visualizer --metadata ./metadata.json && open stats.html",
+    "dev:api-local": "run-p dev:app-local dev:streams-local",
+    "dev:app-local": "./run-api-local.sh",
+    "dev:streams-local": "./run-streams-local.sh"
```

Comment:
Bless you both for separating out the streams from api, so we aren't constantly seeding messages